lib/beaker/host_prebuilt_steps.rb

```diff
@@ -359,6 +364,8 @@ def enable_root_login host, opts
           host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin yes/PermitRootLogin Yes/g' /etc/sshd_config"))
         elsif host['platform'] =~ /freebsd/
           host.exec(Command.new("sudo sed -i -e 's/#PermitRootLogin no/PermitRootLogin yes/g' /etc/ssh/sshd_config"), {:pty => true} )
+        elsif host['platform'] =~ /openbsd/
+          host.exec(Command.new("sudo perl -pi -e 's/^PermitRootLogin no/PermitRootLogin yes/' /etc/ssh/sshd_config"), {:pty => true} )
```

Comment:
Most of the OpenBSD and FreeBSD boxes I've seen have perl installed, as it's a requirement to run VMWare tools and Virtualbox stuff. 

All of the openbsd boxes on VagrantCloud had it that I tested (4 separate) and the two cloud providers that I tried (Cloudsigma and M5Cloud) both had perl on the base image.